### DISCUSSION TITLE
How to compare two streams?

### DISCUSSION THREAD

**Discussion by jignesh-dalal** [paillave/Etl.Net#446] [Q&A]
Input Stream 1:
ID,Name
1,Danny
2,Fred
3,Sam

Input Stream 2:
ID,name
1,Danny
3,Pamela
4,Fernando

Output:
ID,name,Status
1,Danny,Unchanged
2,Fred,Deleted
3,Pamela,Changed
4,Fernando,New

Can this be achieved using Etl.Net?

**Comment by paillave** [comment]
At the moment, these is no way out of the box to compare two stream. Maybe by using the `ToList` operator for both streams, and then, use the `select` operator that joins two `SingleStream`. Then you could do the comparison. I agree it is not great, but as a matter of a fact, a custom operator that would not consume any memory can be done with not much pb. Let me know if you don't find any workaround.
